components/story-inquiry-page/story-concept-faq/StoryConceptFaq.styles.js

```diff
@@ -0,0 +1,47 @@
+
+import styled from 'styled-components';
+import { min } from '../../../lib/responsive';
+import { colors } from '../../../style/colors';
+import {H2, fontWeights} from '../../../style/typography';
+
+const FaqContainer = styled.div`
```

Comment:
nit: can you indent the styling within each object one indent, code style looks off